src/combineReducers.js

```diff
@@ -121,6 +121,8 @@ export default function combineReducers(reducers) {
     if (process.env.NODE_ENV !== 'production') {
       if (typeof reducers[key] === 'undefined') {
         warning(`No reducer provided for key "${key}"`)
+      } else if (typeof reducers[key] !== 'function') {
+        warning(`"${key}" is not a function`)
```

Comment:
This will probably be confusing by itself. I'd add more context to the message: `The reducer for "${key}" is not a function`

Reply:
Good idea! I've updated the error message to ` The reducer for "${key}" is not a function` format.